### DISCUSSION TITLE
G30 cluster pinout

### DISCUSSION THREAD

**Discussion by Dreed420** [VintageCollector/bmw-g30-dashboard-ets2#1] [Q&A] [answered]
Hello friends, I have a quick question - I found the pinout but I don't know the orientation of the socket - 
can you help with some advice or a picture?

![A71-pinout-1](https://user-images.githubusercontent.com/59982076/213580160-3447d866-96cc-434f-b3c8-c43c03722599.png)
![ffc5cbd843708342df2eb880d8eb](https://user-images.githubusercontent.com/59982076/213580372-80a9a6cf-9092-430a-a339-6d72375e290d.jpg)

**Comment by VintageCollector** [comment, marked as answer]
Here is the pinout: https://imgur.com/eXoVL6V
